packages/dynamic_layouts/example/test/staggered_example_test.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(pdblasi-google): Update `window` usages to new API after 3.9.0 is in stable. https://github.com/flutter/flutter/issues/122912
+// ignore_for_file: deprecated_member_use
```

Comment:
Please don't set whole-file ignores, as this will mask other issues as well. The ignores should be on the specific deprecated usages.

Reply:
Done.